main.tf

```
@@ -0,0 +1,77 @@
+module "label" {
+  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.16.0"
+
+  attributes = var.attributes
+  delimiter  = var.delimiter
+  enabled    = var.enabled
+  name       = "${var.name}-ses"
```

```suggestion
  name       = var.name}
```

why do we need to add `ses` suffix? the name is set to `ses` already

---

As this module will for example be used for `sentry` so it makes sense for `ses` prefix, no?

---

top level modules and test fixtures will provide their own mane variable.
No need to hardcode it in the module since it will not work for some people or use-cases